applications/search/src/pages/api-details-page/informationmodel-reference/informationModel-reference.component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { getTranslateText } from '../../../lib/translateText';
+import localization from '../../../lib/localization';
+
+export const InformationModelReference = props => {
+  const { informationModelReference } = props;
+  if (!informationModelReference) {
+    return null;
+  }
+
+  const id = _.get(informationModelReference, ['id']);
+  const prefLabel = _.get(informationModelReference, ['title']);
+
+  return (
+    <div className="d-flex flex-column list-regular--item mb-4">
+      <a
```

Comment:
Her kan du bruke Link fra 'react-router-dom' for å navigere med react-router.

Reply:
Fixed